util/exec.go

```diff
@@ -21,10 +21,10 @@ import (
 
 // Exec will execute the given binary with the given args, sending all messages
```

Comment:
It saves 4 lines in 3 places, so it's not the best helper ever. Want me to nuke it?